packages/jaeger-ui/src/components/TracePage/TraceTimelineViewer/SpanBar.css

```
@@ -44,6 +44,13 @@ limitations under the License.
   z-index: 1;
 }
 
+.SpanBar--cp {
```

```suggestion
.SpanBar--criticalPath {
```
nit: don't be cryptic